src/test/java/com/galvanize/customer/controllers/CustomerRestControllerTests.java

```diff
@@ -28,41 +36,92 @@
     @Autowired
     CustomerRepository repository;
 
+    Customer c;
+
     @Before
     public void setUp() throws Exception {
-
+        c = new Customer();
+        c.setCustomerId(312L);
```

Comment:
The app does not allow for setting your own id here.  When the row is created, the db will assign a new id through the auto commit.  In the save step you are missing below, you can take the Customer that is returned and save it to a local attribute for testing purposes.